doc/configuration-v3_1-experimental.md

```diff
@@ -112,6 +112,21 @@ The Ignition configuration is a JSON document conforming to the following specif
       * **_name_** (string): the group name of the owner.
     * **target** (string): the target path of the link
     * **_hard_** (boolean): a symbolic link is created if this is false, a hard one if this is true.
+ * **_luks_** (list of objects): the list of luks devices to be created. Every device must have a unique `device` & `name`.
+    * **name** (string): the name of the luks device. Every device must have a unique `name`.
+    * **device** (string): the absolute path to the device. Devices are typically referenced by the `/dev/disk/by-*` symlinks.
+    * **_cipher_** (string): the cipher specification string.
+    * **_hash_** (string): the hash used in LUKS key setup scheme & volume key digest.
+    * **_keyFile_** (string): the contents to use as a key file.
```

Comment:
This probably needs some warnings/docs.  For the root volume, it really doesn't make sense right?  For non-root volumes...hmm, I guess one could store key files in e.g. `/etc/luks/$volumename` or so, i.e. unlocking subsequent volumes is bound to the rootfs binding.  That could be nicer than binding each volume to the tpm2 maybe in that if e.g. one chooses to later add a passphrase *in addition* to the tpm2 binding to allow moving the disk later, one would only need to do it for the root?

(There may be a "how to use keyfiles well" doc upstream but I didn't see one in a brief search)